openapi/preprocess_spec.py

```diff
@@ -205,23 +207,23 @@ def find_replace_ref_recursive(root, ref_name, replace_map):
         if "$ref" in root:
             if root["$ref"] == ref_name:
                 del root["$ref"]
-                for k, v in replace_map.iteritems():
+                for k, v in replace_map.items():
                     if k in root:
                         if k != "description":
                             raise PreprocessingException(
                                 "Cannot inline model %s because of "
                                 "conflicting key %s." % (ref_name, k))
                         continue
                     root[k] = v
-        for k, v in root.iteritems():
+        for k, v in root.items():
             find_replace_ref_recursive(v, ref_name, replace_map)
 
 
 def inline_primitive_models(spec):
     to_remove_models = []
-    for k, v in spec['definitions'].iteritems():
+    for k, v in spec['definitions'].items():
         if "properties" not in v:
-            print("Making primitive mode `%s` inline ..." % k)
+            print(("Making primitive mode `%s` inline ..." % k))
```

Comment:
You have another double parentheses here :) Please squash all of your commits into one after this fix.